src/transaction/payment.ts

```diff
@@ -9,7 +9,7 @@ import {Amount, Adjustment, MaxAdjustment,
   MinAdjustment, Memo} from '../common/types/objects'
 
 
-  export type Payment = {
+export type Payment = {
```

Comment:
I think this should be an `interface` rather than a `type`

Reply:
+1, I can change that here